examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1537 @@
+      ]
+     },
+     "execution_count": 51,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Listing all feature services using remote registry client\n",
+    "registry_feature_store_client.list_feature_services()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a074f5bc-1d1b-4ad1-9b08-b3aea58761e8",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Stopping the online, registry server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "id": "d8f353e0-3874-49f7-af82-bd79396eb6d8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/opt/app-root/src/feast/examples/rhoai-quickstart/my_feast_project/feature_repo\n",
+      "1001130+   17522   16173  0 18:03 ?        00:00:04 /opt/app-root/bin/python3.9 /opt/app-root/bin/feast serve\n",
+      "1001130+   17570   17522  0 18:03 ?        00:00:00 /opt/app-root/bin/python3.9 /opt/app-root/bin/feast serve\n",
+      "1001130+   17902   16173  0 18:07 ?        00:00:04 /opt/app-root/bin/python3.9 /opt/app-root/bin/feast serve_registry\n",
+      "1001130+   18481   18438  0 18:15 ?        00:00:00 grep feast serve\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%sh\n",
+    "# checking if the registry server and online server process is already running.\n",
+    "pwd\n",
+    "ps -ef | grep 'feast serve'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "id": "8c84142d-b835-440e-9177-9b8bfe00768f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "remote online and registry server has been stopped.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[2024-09-24 18:16:00 +0000] [17522] [INFO] Handling signal: term\n",
+      "[2024-09-24 18:16:00 +0000] [17570] [INFO] Shutting down\n",
+      "[2024-09-24 18:16:00 +0000] [17570] [INFO] Error while closing socket [Errno 9] Bad file descriptor\n",
+      "[2024-09-24 18:16:00 +0000] [17570] [INFO] Waiting for application shutdown.\n",
+      "[2024-09-24 18:16:00 +0000] [17570] [INFO] Application shutdown complete.\n",
+      "[2024-09-24 18:16:00 +0000] [17570] [INFO] Finished server process [17570]\n",
+      "[2024-09-24 18:16:00 +0000] [17522] [ERROR] Worker (pid:17570) was sent SIGTERM!\n",
+      "[2024-09-24 18:16:00 +0000] [17522] [INFO] Shutting down: Master\n"
+     ]
+    }
+   ],
+   "source": [
+    "feast_online_server_process.terminate()  # Stop the remote Feast online server\n",
```

Comment:
I thought this was interesting (using Popen and then terminate). Is there a more graceful way to stop the feast server? Something like `feast serve stop`). I guess if there isn't, then this works.

Reply:
And it's good that we are cleaning up processes.

Reply:
We don't have the CLI to stop the server probably if `feast serve` can track the underlying process then may be we should be able to implement it. Also the reason we are using `subprocess` is - if we directly start servers using feast cli such as `feast serve` then we can't execute any other cells as the thread is occupied by the feast server.